pilot/cmd/pilot-agent/main.go

```diff
@@ -302,7 +302,7 @@ var (
 
 			opts := make(map[string]interface{})
 			if sdsEnabled {
-				opts["sds_uds_path"] = sdsUdsPathVar.Get()
+				opts["sds_uds_path"] = "unix:" + sdsUdsPathVar.Get()
```

Comment:
How is `SDS_UDS_PATH` set? as far as i can tell there is no installer support for it.

Doesn't this need to end up looking like `unix:///var/run/sds/uds_path` not `unix:/var/run/sds/uds_path` as it would not?

Reply:
I think currently it's just using the default value.

Reply:
Yes, the path is unix:/var/run/sds/uds_path, and it works in my test.